lib/Driver/ToolChains.cpp

```diff
@@ -172,8 +172,10 @@ static void addCommonFrontendArgs(const ToolChain &TC, const OutputInfo &OI,
     arguments.push_back(inputArgs.MakeArgString(OI.SDKPath));
   }
 
+  inputArgs.AddAllArgs(arguments, options::OPT_e, options::OPT_M);
   inputArgs.AddAllArgs(arguments, options::OPT_I);
   inputArgs.AddAllArgs(arguments, options::OPT_F, options::OPT_Fsystem);
+  inputArgs.AddAllArgs(arguments, options::OPT_import_module);
```

Comment:
This one's no longer necessary now that this is not being promoted to a driver option.

Reply:
You know, I actually checked that because I thought I had added it, but the Xcode blame display said it was from an old commit. I looked again just now and it still said that, but I hit Cmd-S and the blame shifted. I wonder what happened.